src/api/utils.js

```diff
@@ -25,6 +26,51 @@ export const formatComment = (comment: Comment): Comment => {
     return formattedComment;
 };
 
+export const getUAAQueryParams = (fileID: string, activityTypes?: UAAActivityTypes[], enableReplies?: boolean) => {
+    const baseEndpoint = `/file_activities?file_id=${fileID}`;
+    const hasFilteredActivityTypes = !!activityTypes && !!activityTypes.length;
+
+    return `${baseEndpoint}${hasFilteredActivityTypes ? `&activity_types=${activityTypes.join()}` : ''}${
+        enableReplies ? '&enable_replies=true&reply_limit=1' : ''
+    }`;
+};
+
+export const parseFileActivitiesResponseForFeed = (data?: UAAFileActivity[]) => {
```

Comment:
Good point, think it could go either way since it is a utils function, but since it's very specific to the FileActivities API currently, I think it's good to have it there